common/lib/xmodule/xmodule/js/src/capa/display.coffee

```diff
@@ -409,7 +424,7 @@ class @Problem
   gentle_alert: (msg) =>
     if @el.find('.capa_alert').length
       @el.find('.capa_alert').remove()
-    alert_elem = "<div class='capa_alert'>" + msg + "</div>"
+    alert_elem = "<div class='capa_alert capa_alert_save'><span class='icon fa fa-floppy-o'></span>" + msg + "</div>"
```

Comment:
We won't want to check in this change. This is the HTML for save? We will want to reuse lms/templates/problem_notifications.html in some way-- perhaps the HTML is always present but has display none/hidden, and that is what is toggled here?

In any case, the story to implement this functionality is https://openedx.atlassian.net/browse/TNL-4935